apps/web/build/fetchConfiguration.ts

```diff
@@ -72,13 +73,13 @@ const convertTemporaryToPermanentEnvironment = () => {
 };
 
 const fetchConfiguration = async () => {
-  if (environmentMap.DISABLE_SETTINGS_FETCH === 'undefined' || environmentMap.DISABLE_SETTINGS_FETCH === '0') {
+  if (environmentMap.DISABLE_SETTINGS_FETCH === undefined || environmentMap.DISABLE_SETTINGS_FETCH === '1') {
```

Comment:
Fixing both cases, first case datatype is undefined not 'undefined' as string.
Second case, when the DISABLE_SETTINGS_FETCH is true => No fetching, so we throw the warn.